### PR TITLE
build with ncdfgenerator

### PR DESCRIPTION
**Pull request by danfruehauf**
@julian1 I didn't test if it works. However it does include all the jars.

### CODE REVIEW

**Comment by danfruehauf** [comment]
Corresponding submodule branch:
https://github.com/aodn/ncdfgenerator/tree/geoserver2_fix_pom (forked from geoserver2)

**Comment by danfruehauf** [comment]
OK, tested that it works. WPS is enabled and the service is published in `GetCapabilities`.

**Comment by julian1** [comment]
I'm reviewing.

**Comment by julian1** [comment]
@danfruehauf This successfully creates the WPS service. But the service fails when called because the ncdfgenerator dependencies such as the ucar netcdf jar are not being pulled in.

**Comment by julian1** [comment]
hmm hang on, let me try again.

**Comment by danfruehauf** [comment]
@julian1 didn't update here - but it does work.

Another thing we need to do, when running geoserver we need to specify `-Djava.awt.headless=true` otherwise I get:
```
java.lang.NoClassDefFoundError: Could not initialize class sun.awt.X11GraphicsEnvironment
```
